tests/integration/commands/declare_cairo1_command_test.py

```diff
@@ -46,6 +46,16 @@ def compile_contract_to_casm_from_contract_name(
             assert compiled is not None
             return compiled
```

Comment:
Are this and method above still used by the test? If not, parts of this mock could be removed.